[plugin.audio.tripler] 2.0.0

molzy opened the pull request:
### Description
I have rewritten an [old Python 2 plugin](https://github.com/DamonToumbourou/plugin.audio.tripler) for the Australian radio station [Triple R](https://www.rrr.org.au/). This rewritten plugin works with the latest versions of Kodi, and uses the radio station's current website to access the live feed, as well as prior segments and episodes.

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

basrieter commented:
Please fix the add-on checker errors before we can review your PR.

molzy commented:
Thanks! I have attempted to resolve the issues that were logged by the addon checker during the first pass.

basrieter commented on plugin.audio.tripler/addon.xml:
```diff
@@ -0,0 +1,32 @@
+		<platform>all</platform>
+		<language>en</language>
+		<summary lang="en">Listen to podcasts avaliable from the Triple R: Independent Melbourne Radio station.</summary>
+		<description lang="en">For more than 40 years Triple R has shaped and inspired the culture of Melbourne. Since its inception as an educational broadcaster in 1976, Triple R has become Australia's most influential community radio station with nearly 21,000 paid subscribers and broadcasting live to over 1,000,000 listeners per month across FM and digital (DAB+ digital radio, podcasts and online).
```
Please use `en_GB` instead of `en` in the summary & description.

basrieter commented on plugin.audio.tripler/addon.xml:
```diff
@@ -0,0 +1,32 @@
+		<summary lang="en">Listen to podcasts avaliable from the Triple R: Independent Melbourne Radio station.</summary>
+		<description lang="en">For more than 40 years Triple R has shaped and inspired the culture of Melbourne. Since its inception as an educational broadcaster in 1976, Triple R has become Australia's most influential community radio station with nearly 21,000 paid subscribers and broadcasting live to over 1,000,000 listeners per month across FM and digital (DAB+ digital radio, podcasts and online).
+Broadcasting on 102.7FM and 3RRR Digital, the Triple R grid houses over 60 diverse programs. Music shows cover every genre imaginable from pop to punk rock, from R&amp;B and electro to jazz, hip hop, country and metal. Specialist talks programs delve into topics as varied as the environment, human rights, politics, medical issues, gardening, cultural ventures and local interests.</description>
+		<license>The MIT License (MIT)</license>
```
Please use the SPDX identifier instead of the full name: https://spdx.org/licenses/

basrieter reviewed (changes requested):
Please see my comments.

molzy commented:
I have made the requested changes, and changed a few details (plugin name, contact email). I have also refactored the code slightly, and fixed an untranslated string.
